[2024-12-19T07:18:10.218+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-12-19T07:18:10.233+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.third_task scheduled__2021-10-06T00:00:00+00:00 [queued]>
[2024-12-19T07:18:10.244+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.third_task scheduled__2021-10-06T00:00:00+00:00 [queued]>
[2024-12-19T07:18:10.245+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 6
[2024-12-19T07:18:10.261+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): third_task> on 2021-10-06 00:00:00+00:00
[2024-12-19T07:18:10.265+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=9139) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-12-19T07:18:10.266+0000] {standard_task_runner.py:72} INFO - Started process 9160 to run task
[2024-12-19T07:18:10.266+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'third_task', 'scheduled__2021-10-06T00:00:00+00:00', '--job-id', '1688', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmp0n_odgs9']
[2024-12-19T07:18:10.269+0000] {standard_task_runner.py:105} INFO - Job 1688: Subtask third_task
[2024-12-19T07:18:10.306+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_v2.third_task scheduled__2021-10-06T00:00:00+00:00 [running]> on host 13c961a72918
[2024-12-19T07:18:10.353+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='code2j' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='third_task' AIRFLOW_CTX_EXECUTION_DATE='2021-10-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-10-06T00:00:00+00:00'
[2024-12-19T07:18:10.354+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-12-19T07:18:10.362+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2024-12-19T07:18:10.363+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task3 and will be running after task1']
[2024-12-19T07:18:10.367+0000] {subprocess.py:99} INFO - Output:
[2024-12-19T07:18:10.368+0000] {subprocess.py:106} INFO - hey, I am task3 and will be running after task1
[2024-12-19T07:18:10.369+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2024-12-19T07:18:10.383+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-12-19T07:18:10.384+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=third_task, run_id=scheduled__2021-10-06T00:00:00+00:00, execution_date=20211006T000000, start_date=20241219T071810, end_date=20241219T071810
[2024-12-19T07:18:10.415+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-12-19T07:18:10.421+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-12-19T07:18:10.423+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
